community/eigen/APKBUILD

```diff
@@ -1,36 +1,49 @@
 # Contributor: Bradley J Chambers <brad.chambers@gmail.com>
 # Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+# Contributor: stef <l0ls0fo2i@ctrlc.hu>
 pkgname=eigen
 pkgver=3.3.7
-pkgrel=0
+pkgrel=2
 pkgdesc="Eigen is a C++ template library for linear algebra"
 url="http://eigen.tuxfamily.org/index.php?title=Main_Page"
 arch="noarch"
-options="!check"  # Headers-only: no tests are possible.
 license="MPL-2.0"
 depends=""
-makedepends=""
+options="!check"  # checks take a long time, and tend to fail randomly
+makedepends="cmake openblas-dev suitesparse-dev mpfr-dev fftw-dev boost-dev gmp-dev glu-dev freeglut-dev"
 install=""
 subpackages="$pkgname-dev $pkgname-doc"
-source="$pkgname-$pkgver.tar.gz::http://bitbucket.org/eigen/$pkgname/get/$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::http://bitbucket.org/eigen/$pkgname/get/$pkgver.tar.gz
+	fftw-test-fix.patch"
 builddir="$srcdir/$pkgname-$pkgname"
 
 build() {
-        return 0
+  cd "$builddir"
```

Comment:
No need to do this, abuild does that for you

Reply:
historical remnant. removed.